test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -585,6 +671,10 @@ def func(x, y):
             func, (torch.tensor([1]), torch.randn(3, 4))
         )
 
+    @skip_model_type_is_exported_program_test(
+        "Unsupported: {'call_function': ['<built-in function ge>', 'aten._assert_async.msg', '<built-in function le>']}."
```

Comment:
Have you updated your onnx-script to the same version as we have on CI right now. It should cover built-in function ge and le...

Reply:
good catch :)
this test was disabled before the fix was merged and I never re-enabled it

Reply:
Now the error changed to 
`torch.onnx._internal.diagnostics.infra.context.RuntimeErrorWithDiagnostic: Unsupported FX nodes: {'call_function': ['aten._assert_async.msg']}.`